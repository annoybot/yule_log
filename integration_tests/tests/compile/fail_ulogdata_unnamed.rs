use yule_log::ULogData;

#[derive(ULogData)]
pub struct TupleStruct(u64, f32);

fn main() {}
