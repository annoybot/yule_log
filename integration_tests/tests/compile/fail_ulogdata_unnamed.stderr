error: ULogData requires named struct fields.
 --> tests/compile/fail_ulogdata_unnamed.rs:4:24
  |
4 | pub struct TupleStruct(u64, f32);
  |                        ^^^
